protocol/Cargo.toml

```diff
@@ -55,12 +55,21 @@ cosmwasm-std = { version = "1.5", default-features = false }
 astroport = { version = "2.9", default-features = false }
 osmosis-std = { version = "0.22", default-features = false }
 
+[profile.dev_no_debug_assertions]
+inherits = "dev"
+debug-assertions = false
+
 [profile.release]
 opt-level = "s"
-debug = false
 strip = "symbols"
 rpath = false
 lto = true
-debug-assertions = false
 panic = "abort"
 overflow-checks = true
+
+[profile.test_nets_release]
+inherits = "release"
+debug-assertions = true
+
+[profile.production_nets_release]
+inherits = "release"
```

Comment:
[here and at platform] let's explicitly state debug and debug-assertions to false